src/index.html

```diff
@@ -5,9 +5,37 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img src="./images/imac.2690641a.jpeg" alt="apple">
+      </div>
+      <div class="card__title">
+        <p>
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+      </div>
+      <div class="card__description">
+        <p>Product code: 195434</p>
+      </div>
+      <div class="card__review review">
+        <img src="./images/Stars.png" alt="start">
+        <p class="review__5">Reviews: 5</p>
+      </div>
+      <div class="card__price price">
+        <p class="price__price">Price:</p>
+        <h3 class="price__dollar">$2,199</h3>
+      </div>
+      <a
+      href="#"
+      class="card__btn"
+      data-qa="hover">Buy</a>
```

Comment:
```suggestion
      <a
        href="#"
        class="card__btn"
        data-qa="hover"
      >
        Buy
      </a>
```
Checklist:
14. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.